website/client/src/router/index.js

```diff
@@ -6,6 +6,8 @@ import handleRedirect from './handleRedirect';
 
 import ParentPage from '@/components/parentPage';
 
+// NOTE: when adding a page make sure to implement setTitle
+
 // Static Pages
 const StaticWrapper = () => import(/* webpackChunkName: "entry" */'@/components/static/staticWrapper');
 const HomePage = () => import(/* webpackChunkName: "entry" */'@/components/static/home');
```

Comment:
I think the home is not being handled by the PR, we should still have a call to `setTitle` to make sure if you go there from a different page it'll reset to simply `Habitica`

Reply:
Do you mean on the /static/home page? For me, it changes correctly. I'm not sure exactly how it is working but when I go to the home page it updates to the title tag in index.html. Should I set it anyways to be more direct?

Reply:
you're right that right now it's not possible to go to that page without reloading but just in case we add the ability in future it'd be nice to have it set explicitly, in which case `Habitica - Gamify Your Life` is the title to use